Formula/gradle.rb

```diff
@@ -4,16 +4,22 @@ class Gradle < Formula
   url "https://services.gradle.org/distributions/gradle-4.0.1-all.zip"
   sha256 "8a8005633be0ca38a206f2590445685683452a0b98a5d8ffd5b8413be09bf998"
 
+  devel do
+    url "https://services.gradle.org/distributions/gradle-4.1-rc-1-all.zip"
```

Comment:
The version detected is `4.1`. Please add
```
version "4.1-rc1"
```
so that this gets treated correctly.